### ISSUE TITLE
Mods that worked with OG QMod manager don't work with QMod Manager Reloaded

### ISSUE THREAD

**Issue by Marianna-Milorzab** [p1xel8ted/GraveyardKeeper#14]
**Mod/tool**
- Mod/tool: OG QMod
- Version: v1.0.8193.37173

**Issue**
- I can't make mods other than yours work. They worked fine with OG QMod Manager, now they don't with QMod Manager Reloaded (while yours work just fine) and I tried many different ways of installation to fix that myself but no luck so far. All of the "old" mods have loading order set as -1 in QMod Manager Reloaded. 
I've noticed on the [screen in nexus](https://staticdelivery.nexusmods.com/mods/2696/images/40/40-1652854998-243657950.png) that you have such mods on your list and they have a proper integer in the loading order column (e.g. More Storage Fixed, Alchemy Research), so I guess that's a desirable state for me as well.

**Screenshots/Video**
![image](https://user-images.githubusercontent.com/78551038/173232630-1eb321cd-4c1b-421e-9ddb-6706e24170ea.png)

**Desktop:**
- Windows 10
- Windows language: English
- Game language : English
- Game platform: Gog

**Additional context**
- I know that on the Nexus the "Intallation" part refers to migrating from OG QMod Manager:

> ** If you're moving across from the OG QMod Manager, you will need to verify files/restore a clean Assembly-Csharp.dll as the loader is injected in a different (earlier) location with a different file name. If you verify after installing QMod Reload Setup, run the installer again to place the needed files back. **

But I can't really say that I get that part. 

I don't think it's a bug, I think it's just my incompetence (but I don't now how to change the label). Could you help me with making my old mods work again and maybe consider more direct instructions how to make that in the documentation?

- qmod_reloaded_log.txt:
```
Sprint added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Sprint added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Extracted D:\Downloads\More Storage Fixed-19-2-4-1638331200.zip.
More Storage Fixed added.
Sprint added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Mod patch injected: System.Void QModReloaded.QModLoader::Patch() inserted into System.Void LogoScene::Awake()
More Storage Fixed added.
Sprint added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\Exhaust-less v7.1-42-7-1-1654905810.zip.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\Fog Be Gone v3 (Harmony2)-46-3-1654180711.zip.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\I Neeeed Sticks v1.2-56-1-2-1654928704.zip.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\Longer Days v3-53-3-1654780368.zip.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\Misc. Bits and Bobs 1.2 (Re-up)-55-1-2-1654932489.zip.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\New Game At Bottom v2 (Harmony2)-43-2-1654180911.zip.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\No Time For Fishing v3 (Harmony2)-44-2-1654180858.zip.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\Queue Everything v1.1-57-1-1-1654930486.zip.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\Save Now v7-41-7-1654929590.zip.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\Thoughtful Reminders v2 (Harmony2)-52-2-1654180334.zip.
Exhaust-less added.
Fog, Be Gone! added.
I Neeeed Sticks! added.
Longer Days added.
Misc. Bits & Bobs added.
More Storage Fixed added.
New Game At Bottom added.
No Time For Fishing! added.
Queue Everything!* added.
Save Now! added.
Sprint added.
Thoughtful Reminders added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Exhaust-less added.
Fog, Be Gone! added.
I Neeeed Sticks! added.
Longer Days added.
Misc. Bits & Bobs added.
More Storage Fixed added.
New Game At Bottom added.
No Time For Fishing! added.
Queue Everything!* added.
Save Now! added.
Sprint added.
Thoughtful Reminders added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Assembly-CSharp.dll has been patched, (otherwise you wouldn't see this message.
Patch method called. Attempting to load mods.
Invoking the specified EntryMethod MoreStorage.Entry.Patch failed for MoreStorage
Invoking the specified EntryMethod Sprint.MainPatcher.Patch failed for Sprint
Load order: 2, successfully invoked Thoughtful Reminders entry method.
Load order: 3, successfully invoked I Neeeed Sticks! entry method.
Load order: 6, successfully invoked Misc. Bits & Bobs entry method.
Load order: 10, successfully invoked Longer Days entry method.
Load order: 11, successfully invoked No Time For Fishing! entry method.
Load order: 13, successfully invoked Exhaust-less entry method.
Load order: 14, successfully invoked New Game At Bottom entry method.
Load order: 15, successfully invoked Save Now! entry method.
Load order: 18, successfully invoked Queue Everything!* entry method.
Load order: 19, successfully invoked Fog, Be Gone! entry method.
Exhaust-less added.
Fog, Be Gone! added.
I Neeeed Sticks! added.
Longer Days added.
Misc. Bits & Bobs added.
More Storage Fixed added.
New Game At Bottom added.
No Time For Fishing! added.
Queue Everything!* added.
Save Now! added.
Sprint added.
Thoughtful Reminders added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\No Intros v2 (Harmony2)-47-2-1654180667.zip.
Exhaust-less added.
Fog, Be Gone! added.
I Neeeed Sticks! added.
Longer Days added.
Misc. Bits & Bobs added.
More Storage Fixed added.
New Game At Bottom added.
No Intros added.
No Time For Fishing! added.
Queue Everything!* added.
Save Now! added.
Sprint added.
Thoughtful Reminders added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Exhaust-less added.
Fog, Be Gone! added.
I Neeeed Sticks! added.
Longer Days added.
Misc. Bits & Bobs added.
New Game At Bottom added.
No Intros added.
No Time For Fishing! added.
Queue Everything!* added.
Save Now! added.
Sprint added.
Thoughtful Reminders added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Res Modifier (GetCurrentResolutions): Obtained current max resolution.
Assembly-CSharp.dll has been patched, (otherwise you wouldn't see this message.
Patch method called. Attempting to load mods.
Invoking the specified EntryMethod Sprint.MainPatcher.Patch failed for Sprint
Load order: 2, successfully invoked Thoughtful Reminders entry method.
Load order: 3, successfully invoked I Neeeed Sticks! entry method.
Load order: 6, successfully invoked Misc. Bits & Bobs entry method.
Load order: 10, successfully invoked Longer Days entry method.
Load order: 11, successfully invoked No Time For Fishing! entry method.
Load order: 13, successfully invoked Exhaust-less entry method.
Load order: 14, successfully invoked New Game At Bottom entry method.
Load order: 15, successfully invoked Save Now! entry method.
Load order: 17, successfully invoked No Intros entry method.
Load order: 18, successfully invoked Queue Everything!* entry method.
Load order: 19, successfully invoked Fog, Be Gone! entry method.
Exhaust-less added.
Fog, Be Gone! added.
I Neeeed Sticks! added.
Longer Days added.
Misc. Bits & Bobs added.
New Game At Bottom added.
No Intros added.
No Time For Fishing! added.
Queue Everything!* added.
Save Now! added.
Sprint added.
Thoughtful Reminders added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Extracted D:\Downloads\p1xel8ted graveyard keeper mods\Auto Loot Heavies v3a (Harmony2)-51-3-1654181097.zip.
Auto-Loot Heavies...Sort Of added.
Exhaust-less added.
Fog, Be Gone! added.
I Neeeed Sticks! added.
Longer Days added.
Misc. Bits & Bobs added.
New Game At Bottom added.
No Intros added.
No Time For Fishing! added.
Queue Everything!* added.
Save Now! added.
Sprint added.
Thoughtful Reminders added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Extracted D:\Downloads\More Storage Fixed-19-2-4-1638331200.zip.
Auto-Loot Heavies...Sort Of added.
Exhaust-less added.
Fog, Be Gone! added.
I Neeeed Sticks! added.
Longer Days added.
Misc. Bits & Bobs added.
More Storage Fixed added.
New Game At Bottom added.
No Intros added.
No Time For Fishing! added.
Queue Everything!* added.
Save Now! added.
Sprint added.
Thoughtful Reminders added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
Auto-Loot Heavies...Sort Of added.
Exhaust-less added.
Fog, Be Gone! added.
I Neeeed Sticks! added.
Longer Days added.
Misc. Bits & Bobs added.
More Storage Fixed added.
New Game At Bottom added.
No Intros added.
No Time For Fishing! added.
Queue Everything!* added.
Save Now! added.
Sprint added.
Thoughtful Reminders added.
All mods with an entry point added. This doesn't mean they'll load correctly or function if they do load.
```

**Comment by p1xel8ted** [comment]
Hi Marianna,

If you downloaded the mods that aren't mine from Nexus directly they won't work due to the updated patch code. I have converted every mod I can and they are available on this repo. 

https://github.com/p1xel8ted/GraveyardKeeper/tree/main/Harmony1-to-2-Mods/DOWNLOAD%20THESE

What mods were you having trouble with? The load order doesn't matter to get them loaded, if it's a negative number it will just load them before the positive numbers. You can drag and drop to change the order.

**Comment by Marianna-Milorzab** [comment]
Ok, no wonder it didn't work, as I had used original nexus files. Now it generally looks fine, thanks for help, p1xel8ted.

As for not working mods on my sidenow, the list is:
- "Sprint" - but it isn't an issue for me as "Not a Keeper's Speed" works fine, and I think it's  a better option to pick.
- "Always Show Experience Bar" - no biggie as it's just a nice extra GUI feature. I've noticed that original Nexus zip contains dll file from 10/06/2022 whereas yours dll file is from 02/06/2022; so maybe there's some update needed on your side, can't really say (I'm unable to check "Alchemy research" at the moment as I haven't unlock alchemy workbench yet but it also has an original dll file from 10/06/2022 whereas yours is from 02/06/2022; on the other hand, MaxButton from the same creator seems to work fine after swapping original Nexus dll file from 10/06/2022 with yours from 02/06/2022)
